cwms-data-api/src/main/java/cwms/cda/data/dao/location/kind/OutletDao.java

```diff
@@ -184,15 +182,21 @@ public void deleteVirtualOutlet(String officeId, String projectId, String virtua
         });
     }
 
-    private Outlet mapToOutlet(PROJECT_STRUCTURE_OBJ_T projectStructure,
+    private Outlet mapToOutlet(PROJECT_STRUCTURE_OBJ_T outlet,
                                List<LocationGroup> groups) {
-        Location location = LocationUtil.getLocation(projectStructure.getSTRUCTURE_LOCATION());
-        CwmsId ratingGroupId = getRatingGroupId(location.getName(), groups);
-        CwmsId projectId = LocationUtil.getLocationIdentifier(projectStructure.getPROJECT_LOCATION_REF());
+        Location location = LocationUtil.getLocation(outlet.getSTRUCTURE_LOCATION());
+        LocationGroup locGroupReturn = getRatingGroupId(location.getName(), groups);
+        CwmsId ratingGroupId = locGroupReturn == null ? null
```

Comment:
Rework this so it's not using ternaries.

Reply:
Switched to Optional